### PR TITLE
fix concurent write error w/signoff

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #
Fix concurent write condition.
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Jougan-0** [comment]
//@dragon-slayer875 @Yashsharma1911

File: mesheryctl/internal/cli/root/model/import.go [hunk 1]
```diff
@@ -175,13 +175,11 @@ func sendToAPI(data []byte, name string, dataType string) error {
...
 	if dataType == "file" {
-		formFile, err = writer.CreateFormFile("file", filepath.Base(name))
+		formFile, _ = writer.CreateFormFile("file", filepath.Base(name))
 	} else {
-		formFile, err = writer.CreateFormField("dir")
+		formFile, _ = writer.CreateFormField("dir")
 	}
-	if err != nil {
 
-	}
```
Comment (Yashsharma1911) [inline mesheryctl/internal/cli/root/model/import.go:184]: Why are we removing the error handling

File: server/helpers/common_registry_logs.go [hunk 2]
```diff
@@ -70,12 +72,14 @@ func HandleError(h v1beta1.Host, en entity.Entity, err error, isModelError bool,
 		if err != nil {
 			handleModelOrRegistrantError(h, entity.Model.Name, err, isModelError, isRegistrantError)
 
+			LogHandler.RegisterAttempts[meshmodel.HostnameToPascalCase(h.Hostname)].mu.Lock()
```
Comment (Yashsharma1911) [inline server/helpers/common_registry_logs.go:75]: Looks good however the logic for v1beta1 and v1alpha two looks mostly similar, you can extract his logic to make sure we are not repeating, something for next PR

**Review by Yashsharma1911 — COMMENTED** [review]
LGTM after this

File: mesheryctl/internal/cli/root/model/import.go (hunk 1 above)
Comment (Jougan-0) [reply mesheryctl/internal/cli/root/model/import.go:184]: In different PR added them.

File: server/helpers/common_registry_logs.go (hunk 2 above)
Comment (Jougan-0) [reply server/helpers/common_registry_logs.go:75]: okay

File: mesheryctl/internal/cli/root/model/import.go (hunk 1 above)
Comment (Jougan-0) [reply mesheryctl/internal/cli/root/model/import.go:184]: When I write support for OCI should be done by today if I get the OCI from Nitish then I'll update this to support it too. Then only I would write the meshkit error for this.

Comment (Jougan-0) [reply mesheryctl/internal/cli/root/model/import.go:184]: @Yashsharma1911

**Pull request merged**